ocfweb/docs/docs/staff/procedures/toner-handling.md

```diff
@@ -0,0 +1,11 @@
+[[!meta title="Recycling Used Toner"]]
+
+OCF goes through toners quite fast, due to the heavy volume of printing from our users. In the past, we've let it accumulate inside the storage room which resulted in accidental toner spillage. The following document contains how to recycle the toners and hope to encourage rapid removal of the toners from OCF space.
```

Comment:
who should be responsible for disposing toner, or ensuring it is disposed?

Reply:
Hmm this, I wanted to maybe delegate this task to opstaff during next contract negotiation, given we are using the UPS pickup service or having it dropped off at the copy center in Moffitt library. I'm icky on delegating this to opstaff to recycle it at the staples since it's far and would ideally require a car.

If staffers were to do it, then we might want to delegate it to Interior Dept.

Reply:
I think it makes more sense for staffers to do it, as long as they actually do it.

You would need to pay opstaff for the time to take it to wherever they need to take it. And that means they wouldn't be at the desk.